modules/heat_conduction/doc/content/source/userobjects/GapFluxModelPressureDependentConduction.md

```diff
@@ -0,0 +1,54 @@
+# GapFluxModelPressureDependentConduction
+
+!syntax description /UserObjects/GapFluxModelPressureDependentConduction
+
+## Description
+
+`GapFluxModelPressureDependentConduction` computes a conductive heat flux across
+a closed gap between two solid bodies as a function of the normal mechanical pressure
+at the interface. The normal contact pressure is included in this calculation as
+a lower-dimensional Lagrange Multiplier variable. This class requires the use of
+[ModularGapConductanceConstraint.md](ModularGapConductanceConstraint.md).
```

Comment:
You could actually add this object in that base class doc, so that users know this is a possibility. Btw, I think one of those `.md`s doesn't need to be there.